lib/brains/forms.py

```diff
@@ -1,12 +1,26 @@
 from django import forms
 from django.core.exceptions import ObjectDoesNotExist
 
+from lib.brains.models import BraintreeBuyer
 from lib.buyers.models import Buyer
 
 
 class BuyerForm(forms.Form):
     uuid = forms.CharField(max_length=255)
 
+    def clean_uuid(self):
+        data = self.cleaned_data
+
+        try:
+            self.buyer = Buyer.objects.get(uuid=data['uuid'])
+        except ObjectDoesNotExist:
+            raise forms.ValidationError('Buyer does not exist.')
```

Comment:
I don't think this is a good way to return validation errors. I filed https://github.com/mozilla/solitude/issues/405